client/browser/src/integration/github.test.ts

```diff
@@ -155,7 +155,7 @@ describe('GitHub', () => {
         const [token] = await line.$x('//span[text()="CallOption"]')
         await token.hover()
         await driver.findElementWithText('User is hovering over CallOption', {
-            selector: '.hover-overlay__content > p',
+            selector: ' [data-testid="hover-overlay__content"] > p',
```

Comment:
We don't need to use BEM for `data-testid` attributes:

```suggestion
            selector: ' [data-testid="hover-overlay-content"] > p',
```